app/javascript/components/Components/Header.js

```diff
@@ -5,14 +5,10 @@ const Header = ({ sign_out_route }) => (
 
     <div id="header">
 
-        <h3 className="title">PageLender</h3>
+        <a href="/"><h3 className="title">PageLender</h3></a>
```

Comment:
Not sure if you were intentionally using an `<a />` here, but you may want to consider using the `react-router-dom` [Link](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/Link.md) instead.

The main difference is that the anchor tag will reload your entire application where as the react-router-dom `<Link />` will not, so your page won't flicker and you'll keep your state.

Reply:
Yes, thanks for the reminder. Switched those.